connect/eaas/runner/constants.py

```diff
@@ -17,9 +17,11 @@
     EventType.ASSET_CANCEL_REQUEST_PROCESSING: 'process_asset_cancel_request',
     EventType.ASSET_ADJUSTMENT_REQUEST_PROCESSING: 'process_asset_adjustment_request',
     EventType.ASSET_RENEW_REQUEST_PROCESSING: 'process_asset_renew_request',
+    EventType.ASSET_TRANSFER_REQUEST_PROCESSING: 'process_asset_transfer_request',
```

Comment:
[COD-A]
There are no such fields in class `EventType`.
I guess they should be added.